examples/upgradable-example/hardhat.config.ts

```diff
@@ -1,6 +1,8 @@
 import '@matterlabs/hardhat-zksync-solc';
 import '@matterlabs/hardhat-zksync-deploy';
+import '@matterlabs/hardhat-zksync-verify';
```

Comment:
We dont need upgradable changes in this PR, focus for this PR is on chai matcher.